app/javascript/packs/application.js

```diff
@@ -0,0 +1 @@
+import '@rails/ujs'
```

Comment:
I would vote to see if we can do `button_to` everywhere and not do JS

Reply:
I added a commit in 514f5e4 to fix the links to be buttons where needed, but the other tough part is we've also lost the confirm dialogs on submission/click (ex: https://github.com/18F/identity-dashboard/blob/77711675f22b15616c641b6fd7fdc59ffdf9f9a3/app/views/users/edit.html.erb#L11)

Reply:
Can we live without the confirmation dialogues? If so happy to drop UJS, but also kinda want to move on to the js-bundling setup

Reply:
My $0.02 is that we should keep UJS for now to avoid any unexpected regressions - I'm not sure where we're using confirmations or non-GET links elsewhere and it doesn't seem worth it to try and find them by hand.